modules/test/playwright/tests/object-web/objectEntriesLocalized.spec.ts

```diff
@@ -869,3 +870,122 @@ test.describe('Localized object entries are saved correctly', () => {
 		}
 	});
 });
+
+test.describe('Required localized object fields', () => {
+	test('verify that default language id is required', async ({
+		apiHelpers,
+		page,
+		viewObjectEntriesPage,
+	}) => {
+		const objectDefinitionLabel = 'ObjectDefinitionLabel' + getRandomInt();
+		const objectDefinitionName = 'ObjectDefinitionName' + getRandomInt();
+
+		const objectFields: ObjectField[] = [
+			{
+				DBType: ObjectField.DBTypeEnum.String,
+				businessType: ObjectField.BusinessTypeEnum.Text,
+				externalReferenceCode: 'textField',
+				indexed: true,
+				indexedAsKeyword: false,
+				indexedLanguageId: '',
+				label: {en_US: 'textField'},
+				listTypeDefinitionId: 0,
+				localized: true,
+				name: 'textField',
+				required: true,
+				system: false,
+				type: ObjectField.TypeEnum.String,
+			},
+		];
+
+		const objectDefinitionAPIClient =
+			await apiHelpers.buildRestClient(ObjectDefinitionApi);
+
+		const {body: objectDefinition} =
+			await objectDefinitionAPIClient.postObjectDefinition({
+				active: true,
+				enableLocalization: true,
+				label: {
+					en_US: objectDefinitionLabel,
+				},
+				name: objectDefinitionName,
+				objectFields,
+				pluralLabel: {
+					en_US: objectDefinitionLabel,
+				},
+				portlet: true,
+				scope: 'company',
+				status: {
+					code: 0,
+				},
+			});
+
+		apiHelpers.data.push({
+			id: objectDefinition.id,
+			type: 'objectDefinition',
+		});
+
+		const objectEntry = await apiHelpers.objectEntry.postObjectEntry(
+			{
+				defaultLanguageId: 'ca_ES',
+				richTextField: getRandomString(),
+				textField: getRandomString(),
+			},
+			'c/' + objectDefinition.name.toLowerCase() + 's'
+		);
+
+		await viewObjectEntriesPage.goto(objectDefinition.className);
+
+		await page.getByRole('link', {name: String(objectEntry.id)}).click();
+
+		await expect(
+			page.getByRole('button', {name: 'ca-es'}).first()
+		).toBeVisible();
+
+		const translationsDropdownTrigger = page
+			.getByTestId('triggerButton')
+			.first();
+
+		await translationsDropdownTrigger.click();
+
+		const catalanOptions = page.getByTestId(
+			'availableLocalesDropdownca_ES'
+		);
+
+		await expect(
+			catalanOptions.first().locator('.label-item-expand')
+		).toHaveText('default', {ignoreCase: true});
+
+		await catalanOptions.first().locator('.label-item-expand').click();
+
+		await page.getByTestId('visibleChangeInput').fill(getRandomString());
```

Comment:
instead of repeating this locator throughout the test, we can store it a const like this:
`const fieldInput = page.getByTestId('visibleChangeInput');
`

and then, on the places you were calling it, replace it with the created const:
`await fieldInput.fill(getRandomString());`